Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
+def ccdBranch = "CIV-0000-adds2stoken"
```

Comment:
please remove it before merging it with master

Reply:
Yes will surely do. Thanks

Reply:
done